app/mailers/user_mailer.rb

```diff
@@ -391,7 +391,8 @@ def delete_signup_notification(user, challenge_signup)
     I18n.with_locale(Locale.find(@user.preference.preferred_locale).iso) do
       mail(
         to: user.email,
-        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Your sign-up for #{@signup.collection.title} has been deleted"
+        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Your sign-up for #{@signup.collection.title} has been deleted",
+        body: ""
```

Comment:
I just mean I haven't gotten one of these emails when deleting challenge sign-ups in recent memory -- the last time I did it on production was April, and because this email didn't sound familiar, I tested it manually on staging shortly after this PR was submitted. It's possible Google's spam filter is eating my emails or it's been broken for a long time and we failed to notice.